scripts/lib/CIME/check_lockedfiles.py

```diff
@@ -9,6 +9,8 @@
 from CIME.XML.env_batch import EnvBatch
 from CIME.utils import run_cmd_no_fail
 
+logger = logging.getLogger(__name__)
```

Comment:
This doesn't appear to be used anywhere in the file. Is pylint OK with this? If so, then I am too.